### PR TITLE
Neta/add attributes to profile

### PR DESCRIPTION
**Pull request by tyranja**
https://github.com/rubymonsters/speakerinnen_liste/issues/1027
https://github.com/rubymonsters/speakerinnen_liste/issues/999

Add a new copy-right column to the pics table in the database
When a speaker upload a picture she can add a copyright to it (not mandatory)
show the copy-right attribute under the picture in the profile view
Add new attribute: format
Add 2 binary attributes:
    willing_to_travel_for_talk
    willing_to_talk_for_nonprofit
    and make them visible when hovering over an profile

### CODE REVIEW

File: app/controllers/admin/profiles_controller.rb [hunk 1]
```diff
@@ -106,11 +106,14 @@ def profile_params
       :website_en,
       :website_2_en,
       :website_3_en,
-      :profession_en,
```
Comment (tyranja) [inline app/controllers/admin/profiles_controller.rb]: why did you remove profession?

Comment (netagonen) [reply app/controllers/admin/profiles_controller.rb]: misstake...
